src/views/Flash/FlashEditor.tsx

```diff
@@ -84,7 +87,10 @@ function EditorContent({ provider, user, setTitle }: {
     <Textbit.Editable
       yjsEditor={yjsEditor}
       onSpellcheck={async (texts) => {
-        return await spellchecker?.check(texts, locale, session?.accessToken ?? '') ?? []
```

Comment:
The last `?? []` should be unnecessary as `.check()` always return `[]`

Reply:
Removing it makes TS emit an error. I'll see if I can make it work.

Reply:
Pushed new commit to mitigate the error. A bit more verbose now maybe, but maybe clearer?